internnav/model/basemodel/navdp/navdp_policy.py

```diff
@@ -7,7 +7,13 @@
 
 from internnav.configs.model.base_encoders import ModelCfg
 from internnav.configs.trainer.exp import ExpCfg
-from internnav.model.encoder.navdp_backbone import *
+from internnav.model.encoder.navdp_backbone import (
+    LearnablePositionalEncoding,
+    NavDP_ImageGoal_Backbone,
+    NavDP_PixelGoal_Backbone,
+    NavDP_RGBD_Backbone,
```

Comment:
Simplify as (and recommend noting the naming convention):
ImageGoalBackbone,
PixelGoalBackbone
RGBDBackbone